mediacenter/templates/mediacenter/index.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'two-third-third.html' %}
+{% load i18n %}
+
+{% block twothird %}
+    <h2><span class="theme read">{% trans "read" %}</span> {% trans "Medias Center" %}</h2>
+    <div class="grid document-list">
+        {% for document in document_list  %}
+            {% include "mediacenter/document_card.html" %}
```

Comment:
There is no good answer to this point. Everybody should know that, that's all. And if it's really a big loop, use external solutions, as caching, pre-renderer (django-template-preprocessor)...